app/_data/docs_nav_ee_0.31-x.yml

```diff
@@ -0,0 +1,60 @@
+- title: Getting Started
+  items:
+    - text: Introduction
+      url: /getting-started/introduction
+    - text: Licensing
+      url: /getting-started/licensing
+    - text: Accessing Your License
+      url: /getting-started/accessing-your-license
+    - text: Getting Started with the Kong Developer Portal
```

Comment:
"Getting started" seems redundant? "The Kong Developer Portal" would be briefer and still scoped under the above "Getting Started" section. 2cts